sources/Inputs/src/AssemblingDataBase.cpp

```diff
@@ -79,6 +79,15 @@ AssemblingDataBase::AssemblingDataBase(const boost::filesystem::path& assembling
   } catch (const xml::sax::parser::ParserException& e) {
     throw Error(DynModelFileReadError, assemblingFilePath.c_str(), e.what());
   }
+
+  if (startingPointMode == dfl::inputs::Configuration::StartingPointMode::FLAT) {
+    for (std::map<std::string, DynamicAutomaton>::const_iterator dynamicAutomatonsIt = dynamicAutomatons_.begin();
+         dynamicAutomatonsIt != dynamicAutomatons_.end(); ++dynamicAutomatonsIt) {
+      if (dynamicAutomatonsIt->second.lib == "SecondaryVoltageControlSimp") {
```

Comment:
Is this the right way to detect the element is a SVC ?

Reply:
I would propose to add a method "SVCInAssembling" and then use it in Context or Configuration to make this test as it does not belong to the ddb

Reply:
> Is this the right way to detect the element is a SVC ?

There is a constant for the SVC library name